### PR TITLE
Add Alexa ranks to GitHub Templates

### LINKED ISSUE #3200: Update Icon Request template
### Kind of issue 
  - [X] Improvement
  - [ ] Bug
  - [ ] Other, namely:

### Description
I think the template for Icon Requests should be updated.

There should be some guidelines on acceptable icons.
Eg, please lookup the Alexa site ranking, please note icons with rankings over 500K might not be accepted.

Alternatively please mention other metrics you have used to display the popularity of this brand.

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
**Issue:** Fixes #3200

### Description
This PR adds the Alexa rank to the information we request from people looking for or submitting a new icon. The goals with this are, in no particular order:
1. To save contributors, maintainers and reviewers a step in having to look up the rank themselves,
2. Hopefully bring a slight reduction in the number of out of scope requests, and,
3. To point people towards our contributing guidelines when opening an issue in the hopes that they might like to take on the icon themselves.

I've also adjusted the layout of the new issue template to give some points more prominence and added a line to encourage people to leave a comment or a reaction if they find an existing issue/PR for the brand they're looking for.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/icon_request.md [hunk 1]
```diff
@@ -4,17 +4,23 @@ about: Request a new icon for SimpleIcons
...
 **Name:**
 **Website:**
+**Alexa rank:**
+  <!-- The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/
+       Please see our contributing guidelines at
+       https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+       for more details on how we assess a brand's popularity. -->
+
 **Official resources for icon and color:**
   <!-- for example media kits, brand guidelines, SVG files, ...) -->
```
Comment (ericcornelissen) [inline .github/ISSUE_TEMPLATE/icon_request.md:26]: You didn't edit this but could you remove the closing parenthesis on this line now that we're editing this file anyway?

File: .github/ISSUE_TEMPLATE/icon_request.md [hunk 2]
```diff
@@ -4,17 +4,23 @@ about: Request a new icon for SimpleIcons
...
 
-<!-- When requesting a new icon we need information such as: -->
+  When requesting a new icon please provide the following information:
+-->
 **Name:**
 **Website:**
+**Alexa rank:**
+  <!-- The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/
+       Please see our contributing guidelines at
+       https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
```
Comment (ericcornelissen) [inline .github/ISSUE_TEMPLATE/icon_request.md:22]: I'm still not sure if I like having the link here... Would you be open to just omitting it and hope contributors can find it themselves? Or maybe put the link at the bottom/top of the template and refer to the links there?

We could also say there's a link to the contributing guidelines in the sidebar, buy I'm personally not a big fan of that either is it depends on a design choice we have no control over.

Any opinions on this from other @simple-icons/maintainers? If most are fine with the current approach I'm fine with leaving it like this as well :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -8,6 +8,13 @@ https://petershaggynoble.github.io/MDI-Sandbox/simpleicons/preview/
 
 **Issue:**
 
+<!-- OR -->
```
Comment (ericcornelissen) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Would it be an option to just ask everyone to provide both an issue reference and Alexa rank (i.e. removing this comment)?

File: .github/ISSUE_TEMPLATE/icon_request.md (hunk 2 above)
Comment (runxel) [reply .github/ISSUE_TEMPLATE/icon_request.md:22]: I'd say either omit it, or add it at the bottom.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (runxel) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Would save us one click on reviewing! So yes, I'd be in favor :)

File: .github/ISSUE_TEMPLATE/icon_request.md (hunk 1 above)
Comment (PeterShaggyNoble) [reply .github/ISSUE_TEMPLATE/icon_request.md]: Done :+1: I also replaced the ellipsis with "etc." for clarity.

File: .github/ISSUE_TEMPLATE/icon_request.md (hunk 2 above)
Comment (PeterShaggyNoble) [reply .github/ISSUE_TEMPLATE/icon_request.md]: Done :+1: Let me know what ye think of the wording

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (PeterShaggyNoble) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Before I _do_ update, see [this comment](https://github.com/simple-icons/simple-icons/issues/3200#issuecomment-642655069) and those that follow for the reasoning behind including the `<!-- OR -->`. If consensus still feels it's unnecessary, though, I'll remove it.

Comment (ericcornelissen) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: If you are concerned about the Alexa rank already being reported in the issue, then I don't think it hurts to ask contributors to state it again in the PR (as it does indeed safe is one click and some potential search work, though it's usually in the first or second comment).

If you're concerned about another metric being more relevant, perhaps we can change the wording to "Popularity" or something similar?

Comment (PeterShaggyNoble) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: > If you are concerned about the Alexa rank already being reported in the issue, then I don't think it hurts to ask contributors to state it again in the PR
A good point: They can just copy/paste it from the issue - I was clearly overthinking things in that people would go and look it up again!

I've removed the `<!-- OR -->` now. I think we should just run with the  `**Alexa rank:**` label for now, until we've firmed away the cut-off points for some of our other metrics and added them to our guidelines.

File: .github/ISSUE_TEMPLATE/icon_request.md [hunk 4]
```diff
@@ -4,17 +4,25 @@ about: Request a new icon for SimpleIcons
...
+       assess a brand's popularity. -->
 
-<!--
-We won't add non-brand icons or anything related to illegal services.
-If in doubt open an issue and we'll have a look.
--->
+**Official resources for icon and color:**
+  <!-- for example media kits, brand guidelines, SVG files, etc. -->
+  
+<!-- For more deatils on our processes our contributing guidelines
```
Comment (ericcornelissen) [inline .github/ISSUE_TEMPLATE/icon_request.md:27]: Spotted a typo here

```suggestion
<!-- For more details on our processes our contributing guidelines
```

Comment (PeterShaggyNoble) [reply .github/ISSUE_TEMPLATE/icon_request.md]: Oops! Thanks, Eric :+1:

**Comment by ericcornelissen** [comment]
Thanks @PeterShaggyNoble :+1:

**Pull request merged**